app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_posts_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Great job creating a method that updates the likes counter for a post.. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_posts_likes_counter` method every time a new like has been saved to the database, then make the `update_posts_likes_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :

```ruby
# app/models/like.rb
...
     after_save : update_posts_likes_counter
   
     private

    def update_likes_counter
       post.increment!(:likes_counter)
    end
```

Reply:
done